src/nanoarrow/common/array_test.cc

```diff
@@ -3343,7 +3343,50 @@ TEST(ArrayViewTest, ArrayViewTestGetString) {
   TestGetFromBinary<FixedSizeBinaryBuilder>(fixed_size_builder);
 }
 
-template <typename BuilderClass>
+static const void* GetBinaryData(const struct ArrowBufferView* bv) {
```

Comment:
Open to different ideas on how to best template these tests; if constexpr would be really nice here!